backend/package.json

```diff
@@ -20,6 +20,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@koderholic/my-collectives": "^1.5.4",
```

Comment:
You should perhaps use the public version:     "@liquality/my-collectives": "2.0.1",
and then it will be no need to have npmrc file 👍